glacierscript.py

```diff
@@ -3,15 +3,15 @@
 ################################################################################################
 #
 # GlacierScript:  Part of the Glacier Protocol (http://glacierprotocol.org)
-# 
```

Comment:
Commingling trivial changes like this with substantive changes in a single commit makes review unnecessarily difficult. Best to make a separate commit with only the trivial changes next time.

Reply:
Thanks for the response. Understood re pull request separation - I am a novice at this as usually hacking only my own code. Should I re-submit my individual branches (sequential-sign, tails, base-upgrades)?

Reply:
Don't bother re-submitting...they are reasonably-enough segregated already.

Reply:
I think I misunderstood you here...I meant that the commits are segregated. I understand now you are asking about separate PRs, and yes I think separate PRs would be good.